main.tf

```
@@ -1,5 +1,5 @@
 locals {
-  port = var.port == "" ? (var.engine == "aurora-postgresql" ? 5432 : 3306) : var.port
+  port = var.port != null ? var.port : (var.engine == "aurora-postgresql" ? 5432 : 3306)
```

In some places we have it like this:

```suggestion
  port = coalesce(var.port, (var.engine == "aurora-postgresql" ? 5432 : 3306))
```

This will work with an empty string and `null`. You decide.

---

I dig it - thanks!

---

Great.

PS: The big PR by your is for tomorrow :) Have a great week-ends!